### DISCUSSION TITLE
Medley memorabilia

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1666] [General]
In the April 17, 2024 external meeting @MattHeffron showed an awesome LOOPS lapel pin. Matt, can you please share a photo of the pin?

In the same meeting @masinter mentioned an Interlisp-D pen which would be also great if Larry could share a photo of.

**Comment by MattHeffron** [comment]
The colors are darker than originally.
Here it is brightened:
![20240417_164641 adj](https://github.com/Interlisp/medley/assets/8261874/0b560b3b-3c07-43ea-8722-c0f9f085cc75)
And here is the original:
![20240417_164641](https://github.com/Interlisp/medley/assets/8261874/a8bd9494-e63b-4074-9f7f-eda9d76b86f4)
